sample-shrinker.sh

```diff
@@ -90,6 +90,10 @@ Originally created to reduce the stress of streaming multiple simultaneous
     Valid values are '8' and '16'
     Default: ${minimum_bit_depth:-(none)}
 
+-s SAMPLERATE
```

Comment:
The args code doesn't accept `-s`―but it now accepts `-r` and `-R` , which are undocumented

Reply:
Remnant from earlier before I switched to r - great catch thank you!